Assets/Tests/Editor/Integration/CodePatterns/CompileWithRedirectTestBase.cs

```diff
@@ -82,8 +82,7 @@ private static CompileResult CompileCode(string filePath)
         
         protected static string ResolveFullTestFilePath(string relativePath)
         {
-            //TODO: how to resolve test path relative to proj dir for tests??
-            return @$"E:\_src-unity\FastScriptReload\Assets\Tests\{relativePath}";
+            return @$"{Directory.GetCurrentDirectory()}\Assets\Tests\{relativePath}";
```

Comment:
don't think this will work when we consume FSR as a package as it'll not be in your proj dir but somewhere in packages

Reply:
What do you mean? I'm using FSR as a package, and its tests are not shown in Test Runner.

Reply:
hhmm - that's interesting. They are included in code but you're right that they do not show in test runner. Maybe that's by Unity design. Cool - that's one TODO sorted then :)